lib/auth/auth.go

```diff
@@ -276,9 +276,19 @@ var (
 		},
 	)
 
+	registeredAgents = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: teleport.MetricNamespace,
+			Name:      teleport.MetricRegisteredAgents,
+			Help:      "The number of Teleport agents with their version that have connected to the Teleport cluster. After disconnecting, a Teleport agent has a TTL of 10 minutes so this value will include agents that have recently disconnected but have not reached their TTL.",
```

Comment:
I know what you mean here, but is "agent" a term that we consistently use to describe nodes, kube services, db services, etc? I haven't heard them referred to as agents much.

If this is the case, maybe we want to clarify:

`the number of Teleport agents (nodes, app services, etc) that have connected...`

Reply:
Perhaps this is better worded as "Teleport instances" as an instance of Teleport may consists of multiple services but this would only be incremented once for the same Teleport instance running multiple services.